config/anyscale.yaml

```diff
@@ -27,7 +27,7 @@ chat_engine:
   # Since Anyscale's LLM endpoint currently doesn't support function calling, we will use the LastMessageQueryGenerator
   # --------------------------------------------------------------------
   query_builder:
-    type: LastMessageQueryGenerator     # Options: [FunctionCallingQueryGenerator, LastMessageQueryGenerator]
+    type: CondensedQueryGenerator     # Options: [FunctionCallingQueryGenerator, LastMessageQueryGenerator]
```

Comment:
We should list `CondensedQueryGenerator` as an option in the default `config.yaml` as well